src/Reader/Entry/Atom.php

```diff
@@ -70,6 +69,7 @@ public function getAuthors()
             return $this->data['authors'];
         }
 
+        /** @psalm-suppress PossiblyNullReference */
```

Comment:
We should probably use a baseline, instead of suppressing these? Should we perhaps assert that the extension got instantiated instead? (applies to subsequent `->getExtension()` calls too)

Reply:
Perhaps isolating to `private function getAtomExtension()`?

Reply:
I've added private functions for fetching each type of extension in each of the RSS and Atom entry classes now. Each attempts to retrieve the associated extension, tests that it is the correct type (throwing a `RuntimeException` if not), and returns it.